include/eal.h

```diff
@@ -1,21 +1,25 @@
-#ifndef EAL_H
-#define EAL_H
+#ifndef EAL_INCLUDE_EAL_H
+#define EAL_INCLUDE_EAL_H
 
 #include <vector>
-#include <string.h>
+#include <string>
 
 using namespace std;
 
-vector<double> getPosition(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time);
-vector<double> getVelocity(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time, bool interpolation);
+namespace eal {
 
-vector<double> getPosition(vector<double> coeffs, char *interp, double time);
-vector<double> getVelocity(vector<double> coeffs, char *interp, double time, bool interpolation);
+  vector<double> getPosition(vector<vector<double>> coords, vector<double> times, string interp, double time);
+  vector<double> getVelocity(vector<vector<double>> coords, vector<double> times, string interp, double time, bool interpolation);
 
-vector<double> getRotation(char *from, char *to, vector<vector<double>> rotations, vector<double> times, char *interp, double time);
-vector<double> getAngularVelocity(char *from, char *to, vector<vector<double>> rotations, vector<double> times, char *interp, double time);
+  vector<double> getPosition(vector<double> coeffs, string interp, double time);
+  vector<double> getVelocity(vector<double> coeffs, string interp, double time, bool interpolation);
 
-vector<double> getRotation(char *from, char *to, vector<double> coefficients, char *interp, double time);
-vector<double> getAngularVelocity(char *from, char *to, vector<double> coefficients, char *interp, double time);
+  vector<double> getRotation(string from, string to, vector<vector<double>> rotations, vector<double> times, string interp, double time);
```

Comment:
Do we want a `bool interpolation` on this one?

Reply:
I guess I'm not sure what we would do if =false. "Sorry, no rotation data at this point?"

Reply:
I think the getAngularVelocity need bool interpolation options. I don't think this one does.

Reply:
Agreed, that makes more sense.

Reply:
I'll make that change and push it up